src/backends/unicodeplots.jl

```diff
@@ -184,13 +185,16 @@ function addUnicodeSeries!(
         end
     elseif st in (:surface, :wireframe)  # 3D
         colormap = get(series[:extra_kwargs], :colormap, :none)
+        lines = get(series[:extra_kwargs], :lines, st === :wireframe)
+        zscale = get(series[:extra_kwargs], :zscale, :identity)
```

Comment:
Same here, we have `sp[:zaxis][:scale]` as well as `colorbar_scale` or is it about something else?